srv/src/main/java/sap/capire/xtravels/handler/RecalculatePriceHandler.java

```diff
@@ -67,6 +71,20 @@ private void updateTotals(CqnStructuredTypeRef ref) {
         Update.entity(travel).data(Travels.TOTAL_PRICE, totalPrice).hint("@readonly", false));
   }
 
+    @After(event = EVENT_CREATE)
+    void setTotalPriceAfterCreation(Travels travels) {
```

Comment:
I don't think we should showcase a calculated field in an @After handler. You should rather calculate the field in a @Before handler to avoid an additional db roundtrip.